SnapWar/Components/Common/RowView.swift

```diff
@@ -0,0 +1,19 @@
+//
```

Comment:
i don't think this file was added properly to xcode, could you double check? and then also remove the default header pls 😄

Reply:
actually never mind, i can just delete this since we don't use it